src/app/containers/StoryPromo/index.jsx

```diff
@@ -11,13 +11,41 @@ import FigureContainer from '../Figure';
 
 import { ServiceContext } from '../../contexts/ServiceContext';
 import deepGet from '../../lib/utilities/deepGet';
+import createSrcset from '../Image/helpers/srcSet';
+
+const pathToArray = path => path.slice(1).split('/');
+
+const getOriginCode = path => {
```

Comment:
Hey, can you move these two files into a seperate file please, and write input/output assertion tests for them both :)